astropy/cosmology/core.py

```diff
@@ -462,16 +462,16 @@ def clone(self, **kwargs):
             except AttributeError:
                 # We didn't find a property -- complain usefully
                 errstr = "Object did not have property corresponding "\
-                         "to constructor argument '%s'; perhaps it is a "\
+                         "to constructor argument '{}'; perhaps it is a "\
```

Comment:
There are also a lot of codes in Astropy that has `{0}` instead of `{}` that were added back when we support Python 2.6. Not sure if you want to make those consistent as well or not. But they are harmless if left alone. FYI.